sympy/codegen/tests/test_ast.py

```diff
@@ -251,14 +251,14 @@ def test_For():
 
 def test_none():
     assert none.is_Atom
-    assert none == none
+    assert none is none
     class Foo(Token):
         pass
     foo = Foo()
     assert foo != none
-    assert none == None
+    assert none is None
     assert none == NoneToken()
-    assert none.func(*none.args) == none
```

Comment:
`none` is not a singleton so creating a new instance will not pass the `is` check.